packages/ui/client/components/ConsoleOutput.vue

```diff
@@ -0,0 +1,37 @@
+
+<script setup lang="ts">
+import type { File } from '#types'
+
+const props = withDefaults(defineProps<{
+  modelValue?: boolean
+  file?: File
+}>(), {
+  modelValue: false,
+  file: undefined,
+})
+
+defineEmits<{
+  (e: 'update:modelValue', modelValue: boolean): void }
+>()
+
+const consoleOutput = computed(() => props.file?.tasks?.map(i => i?.logs || []).flat() || [])
+</script>
+
+<template>
+  <div w-full>
+    <div
+      flex justify-between cursor-pointer bg-header h-auto all:transition-400 p="x4 y2"
+      @click="$emit('update:modelValue', !modelValue)"
+    >
+      <p> Console output ({{ consoleOutput?.length || 0 }}) </p>
+      <IconButton icon="i-carbon-caret-down" :class="!modelValue ? 'rotate-180' : 'rotate-0'" :disabled="!logs?.length" />
+    </div>
+    <template v-if="modelValue">
+      <div p="x4 y6">
+        <div v-for="log of consoleOutput" :key="log" my-2>
```

Comment:
Idea: Add indication for stdout vs stderr logs

Reply:
Good suggestion, I'm wrapping everything in pre now to have some formatting will add the use of `text-orange` if the `log.type` is `stderr`